index.html

```diff
@@ -308,6 +352,42 @@ <h3 class="language-title">Skills</h3>
       </ul>
       <hr class="footer-line" />
     </footer>
+    <section class="popup-container">
+      <div class="overlay"></div>
+      <div class="popup-contain">
+        <div class="title-plus">
+          <h1 class="popup-title"></h1>
+          <span class="plus">+</span>
+        </div>
+        <ul class="popup-btns">
+          <li class="popup-btn"></li>
+          <li class="popup-btn"></li>
+          <li class="popup-btn"></li>
+          <li class="popup-btn"></li>
+        </ul>
+        <div class="displayPC">
+          <img class="popup-image" src="" />
+          <div class="pdes">
+            <p class="popup-description"></p>
+            <br /><br />
+            <p class="popup-description2"></p>
+            <a href='#' class="see-live"><button class="see-live live-source" type="button">
+              See live
+              <span class="ellipse"
+                ><img src="images/ellipse.png" alt="Ellipse"
+              /></span>
+            </button></a>
+          <a href='#' class="see-source"><button class="see-source live-source" type="button">
+              See source
+              <span class="popgit">
+                <img src="images/popgit.png" alt="github" />
+              </span>
+            </button></a>
```

Comment:
As requested by previous CRs your buttons should be responsive, meaning that when the user gets on them they should be able to see some effect. Kindy improve that as follows 👏  

```bash
button {
 ...
cursor: pointer;
}

.button: hover {
  background-color: ...;
  color: ...;
}
// etc
```
It will help the users to directly know that the button is clickable